projects/igniteui-angular/src/lib/combo/combo.component.spec.ts

```diff
@@ -213,17 +211,18 @@ describe('igxCombo', () => {
             spyOn(mockIconService, 'addSvgIconFromText').and.returnValue(null);
             combo.ngOnInit();
             combo.data = data;
-            spyOn(combo, 'selectItems');
+            mockSelection.select_items.calls.reset();
+            spyOnProperty(combo, 'isRemote').and.returnValue(false);
             combo.writeValue(['EXAMPLE']);
-            expect(combo.selectItems).toHaveBeenCalledTimes(1);
+            expect(mockSelection.select_items).toHaveBeenCalledTimes(1);
 
-            // Calling "SelectItems" through the writeValue accessor should clear the previous values;
+            // Calling "select_items" through the writeValue accessor should clear the previous values;
             // Select items is called with the invalid value and it is written in selection, though no item is selected
             // Controlling the selection is up to the user
-            expect(combo.selectItems).toHaveBeenCalledWith(['EXAMPLE'], true);
+            expect(mockSelection.select_items).toHaveBeenCalledWith(combo.id, ['EXAMPLE'], true);
             combo.writeValue(combo.data[0]);
             // When value key is specified, the item's value key is stored in the selection
-            expect(combo.selectItems).toHaveBeenCalledWith(combo.data[0], true);
+            expect(mockSelection.select_items).toHaveBeenCalledWith(combo.id, [], true);
```

Comment:
@ViktorSlavov I couldn't figure out what the this was initially added for #5569 and I still don't see the point and after a refactor it doesn't seem to test much of anything, outside a broken scenario calling with a single data item.

If you have a scenario this is supposed to cover and it doesn't - let me know

Reply:
I believe this was to verify that no item in the drop-down is marked w/ a selection class when an invalid value is passed to the selection service. Since it's now a pure unit test, there's no need for this scenario. And yeah, it was testing incorrect usage of the control.